### DISCUSSION TITLE
How to handle long running process with kafkaflow?

### DISCUSSION THREAD

**Discussion by dosper7** [Farfetch/kafkaflow#376] [Q&A] [answered]
I'm trying to setup a saga-like pattern with kafkaflow, something similar to [this](https://github.com/rebus-org/Rebus/wiki/Coordinating-stuff-that-happens-over-time). Is there any approach to achieve this with kafkaflow?

**Comment by gsferreira** [comment, marked as answer]
Hi @dosper7!

At the moment KafkaFlow doesn't have a built-in way to create Sagas. However, it's not the first time that the question has popout.
So, maybe it would be a nice feature to consider at some point.
What do you think @filipeesch? Worth opening a Feature Request?
